.github/workflows/ansible-test.yml

```diff
@@ -38,6 +38,8 @@ jobs:
       - name: Convert role to collection format
         run: |
           set -euxo pipefail
+          # Remove to avoid running ansible-test on unrelated file
```

Comment:
Can't this be fixed using [ignores](https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/ignores.html) ?

Reply:
.pandoc_template.html5 is a hidden file that is not present in the resulted collection that we publish to Ansible Galaxy and AH because we remove dotted files anyway, so adding an ignore for it is an overkill

Reply:
I'm fine with that.